otter/integration/tests/test_convergence.py

```diff
@@ -322,7 +410,9 @@ def _choose_random_servers(self, (code, response), n):
         return random.sample(ids, n)
 
     def _delete_those_servers(self, ids, rcs):
-        """Delete each of the servers selected."""
+        """
+        Delete each of the servers selected, and save a list of the
+        ids of the deleted servers."""
```

Comment:
Non-blocking: I'm thinking we should document somewhere all the values that are being saved in the test cases in a central location if it isn't already. (ex. self.removed_ids, n_killed, etc.)